guides/kubernetes.md

```diff
@@ -0,0 +1,136 @@
+# Kubernetes guidance
+
+## Configure NGINX Ingress Controller
+An [Ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) is an application that runs in a Kubernetes cluster and configures an HTTP load balancer according to Ingress resources.
+
+In the case of NGINX, the Ingress controller is deployed in a pod along with
+the load balancer.
+
+### Installation
+The documentation for [NGINX's chart](https://github.com/helm/charts/blob/master/stable/nginx-ingress/README.md) includes details on how to install it.
+
+*TL;DR:*
+
+`helm install stable/nginx-ingress --name nginx-ingress`
+
+### Creating a namespace for a service
+Each service in a cluster will have their own dedicated namespace in each cluster.
+
+This allows logical separation between services as well as the enabling simpler implementation of monitoring, RBAC and stability mechanisms.
+
+## Requirements
+The following are the suggested outcomes of a namespace.
+- `ResourceQuota` resource to limit resource usage
+- `RoleBinding` resource to restrict interaction to delivery team
+
+Example setup instructions are included in the [FFC Kubernetes Configuration](https://github.com/DEFRA/ffc-kubernetes-configuration) repository.
+
+## Install Kubernetes dashboard
+The Kubernetes dashboard is a web-based Kubernetes user interface.
+
+### Installation
+Install the dashboard to a Kubernetes cluster using the `kubectl apply` command specified in the **Deploying the dashboard UI** section in the below link.
+
+https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/
+
+Example:
+`kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml`
+
+#### Create default user and access token
+Follow the guide in the below link.
+https://github.com/kubernetes/dashboard/blob/master/docs/user/access-control/creating-sample-user.md
+
+#### Run dashboard
+1. Run terminal command `kubectl proxy`
+1. Access dashboard at http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/
+
+## Interact with cluster
+`kubectl` is used to interact with the cluster.  In order to use `kubectl` with an FFC cluster, a `kubeconfig` file for the cluster is required.
+
+### Acquiring a Kubeconfig file for a cluster in AKS
+To acquire a Kubeconfig, a subscription Id is needed along with the name of the cluster and the resource group in which it resides.  This information can be acquired via the Azure Portal or from CSC.
+
+`az account set --subscription SUBSCRIPTION_ID`
+
+`az aks get-credentials --resource-group RESOURCE_GROUP --name CLUSTER --file WHERE_TO_SAVE_KUBECONFIG`
+
+**Note** if the file parameter is not passed, the Kubeconfig will be merged with the users default configuration file stored at `~/.kube/config`.
```

Comment:
I know from bitter experience that the aks command can trash your local `~/.kube/config`, if you don't specify a new file. It may be worth adding a warning of the perils here!

Reply:
Yes, very true, I've changed the word `Note` to `Warning` to get greater attention and added a subsequent line highlighting the outcome of the risk.  Very frustrating the Azure docs don't include that argument and don't give any warning that it will modify existing files on your device until it's already done.